src/Sylius/Bundle/ApiBundle/Attribute/OrderTokenValueAware.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Attribute;
+
+#[\Attribute(\Attribute::TARGET_CLASS)]
+final class OrderTokenValueAware
+{
+    public function __construct(public string $constructorArgumentName = 'orderTokenValue')
```

Comment:
Maybe we should extract the default names to consts within the attributes (or better yet, the interface) and reuse that in here and in the context builders;
Right now we have the same data hardcoded in multiple places, and the relation between the interface, attribute and context builder is muddled.

But that's something for another PR.